service/controller/app/v1/resource_set.go

```diff
@@ -158,19 +134,28 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 			return nil, microerror.Mask(err)
 		}
 
-		restConfig, err := kubeConfig.NewRESTConfigForApp(ctx, cr)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-
-		g8sClient, err := versioned.NewForConfig(restConfig)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-
-		k8sClient, err := kubernetes.NewForConfig(restConfig)
-		if err != nil {
-			return nil, microerror.Mask(err)
+		// TODO: remove if statement after in-cluster flag is implemented
```

Comment:
Do we need to track a TODO issue for this or when will it be addressed?

Reply:
After this PR, we are going to apply in-cluster changes. https://github.com/giantswarm/app-operator/pull/36#issuecomment-464761774